site/layouts/partials/sidebar/submenu-begin.html

```diff
@@ -1,5 +1,5 @@
 
-<a href="{{ .href }}" class="list-group-item collapsed" data-toggle="collapse" data-parent="#sidebar" aria-expanded="false" style="z-index: inherit">
+<a href="{{ .href }}" class="list-group-item collapsed" data-toggle="collapse" data-parent="{{ .parent }}" aria-expanded="false" style="z-index: inherit">
```

Comment:
@manbhard , We had this statically set because of a bug that we talked about.  The bug is that in a nested menu with multiple parents, multiple selects (active links) get stuck.  This is happening with your change now, for example select on getting started -> requirements then select on the new submenu item.  You'll see two active, but obviously only one is really displayed in docs content.

Having two active selections under the same root submenu is misleading and incorrect in terms of which item is actually being displayed.